src/sentry/options/store.py

```diff
@@ -94,9 +95,15 @@ def get(self, key, silent=False):
         if result is not None:
             return result
 
-        result = self.get_store(key, silent=silent)
-        if result is not None:
-            return result
+        with sentry_sdk.start_span(op="sentry_options_store.cache_miss") as span:
```

Comment:
I have some concern that we'll generate a large number of spans with this and lose visibility in other parts of the application because our tracing spans will be consumed by these option spans. Could we collect these spans for a subset of production traffic? Like 1-10%?

Reply:
Yeah I agree. I didn't see an obvious way to set a sample rate for a span in our docs, and didn't see any option for this in the documentation either.

In the meantime, I put up a slightly different PR that samples and logs, which we should allow us to use log analytics to query pretty easily.